src/components/UTIL_FormField.component

```diff
@@ -78,7 +82,8 @@
                     || $ObjectType[sObjType].fields[field].createable && sObj['Id']=='') && $ObjectType[sObjType].fields[field].type == 'boolean'}">
         <div class="slds-form-element__control">
             <apex:outputLabel styleClass="slds-checkbox" for="inputCB" id="lblCheckbox">
-                <apex:inputField id="inputCB" value="{!sObj[field]}" label="">
+                <apex:inputField id="inputCB" value="{!sObj[field]}" label=""
+                html-aria-description="{!IF(ISBLANK(ariaDescription), '', ariaDescription)}">
```

Comment:
Is `html-aria-description` set by default?

Reply:
Update to have the default value label if ariaDescription is missing. aria-description is never set before.